doc/source/contributing.rst

```diff
@@ -2,48 +2,50 @@
 Contributing
 ============
 
-The general guidance is available in the `Contributing` topic in the *PyAnsys
-Developer's Guide*. PyPIM follows the PyAnsys `Guidelines and Best Practices`_.
-
+Overall guidance on contributing to a PyAnsys repository appears in the
+`Contributing`_ topic in the *PyAnsys Developer's Guide*. Ensure that you are
+thoroughly familiar with it and all `Guidelines and Best Practices`_
+before attempting to contribute to PyPIM.
+ 
 .. _`Contributing`: https://dev.docs.pyansys.com/overview/contributing.html
 .. _`Guidelines and Best Practices`: https://dev.docs.pyansys.com/guidelines/index.html
 
+The following contribution information is specific to PyPIM.
 
 Cloning the PyPIM Repository
 ----------------------------
+Run this code to clone and install the latest version of PyPIM in development mode:
 
 .. code-block::
     
     git clone https://github.com/pyansys/pypim.git
     cd pypim/
 
-Running the tests
------------------
-
-The test automation relies on `tox`.
+Running Tests
+-------------
+Test automation relies on `tox`.
 
-They are entirely based on mocks and do not require any external software. Run
-the tests with:
+Tests are entirely based on mocks and do not require any external software. Run
+the tests with the following code, where `py38` represents your Python version:
```

Comment:
Can we assume that users knows how to check to see what Python version they have? (I don't know.)